server/public/html/index.html

```diff
@@ -1,43 +1,239 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<!DOCTYPE html>
 <html xmlns="http://www.w3.org/1999/xhtml">
-
 <head>
 	<meta http-equiv="content-type" content="text/html; charset=utf-8" />
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="icon" href="../favicon.png" sizes="230x230" type="image/png">
   <title>UD Annotatrix</title>
 	<link rel="stylesheet" type="text/css" href="../css/bootstrap.min.css" />
-	<link rel="stylesheet" type="text/css" href="../css/font-awesome.min.css" />
+	<!-- <link rel="stylesheet" type="text/css" href="../css/font-awesome.min.css" /> -->
 	<link rel="stylesheet" type="text/css" href="../css/style.css" />
+	<style>
+     body {
+       padding-top: 100px;
+     }
+		 .actmode{
```

Comment:
Why the change in indentation?  Are you using spaces or tabs?

Reply:
I use Atom, and it seems to use tabs by default, I switch it to spaces.